setup.py

```diff
@@ -3,46 +3,44 @@
 import os
 from setuptools import find_packages, setup
 
-# Where are we?
-IS_SANDBOX = "sandbox" in os.getenv("JUPYTER_IMAGE", default="")
```

Comment:
This causes more problems than it solves, and can lead to some really cryptic bugs.